package.json

```diff
@@ -329,7 +332,7 @@
     "copy-webpack-plugin": "11.0.0",
     "css-loader": "^6.8.1",
     "css-minimizer-webpack-plugin": "^5.0.1",
-    "cypress": "12.17.4",
+    "cypress": "13.2.0",
```

Comment:
Is it neccesary to update cypress for this PR?

Reply:
Not at all, was added during the problems to run cypress against chrome. I'll update to have the same as development.